src/app/chat/view.nim

```diff
@@ -103,7 +103,7 @@ QtObject:
     read = getMessageList
     notify = activeChannelChanged
 
-  proc pushChatItem*(self: ChatsView, chatItem: ChatItem) =
+  proc pushChatItem*(self: ChatsView, chatItem: var Chat) =
```

Comment:
I don't think we're mutating `chatItem` here so `chatItem: Chat` should do the trick